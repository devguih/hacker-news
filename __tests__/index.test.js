/* eslint-env jest */

import { shallow } from 'enzyme'
import React from 'react'
import renderer from 'react-test-renderer'

import List from '../pages/list.js'

describe('With Enzyme', () => {
  it('List shows', () => {
  })
})

describe('With Snapshot Testing', () => {
  it('List shows', () => {
    const items = []
    const component = renderer.create(<List items={items} />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
