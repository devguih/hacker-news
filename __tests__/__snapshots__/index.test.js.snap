// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing List shows 1`] = `
<div>
  <div>
    <div>
      <div
        className="navbar"
      />
      <nav
        className="navbar is-link is-fixed-top"
      >
        <div
          className="navbar-brand"
        >
          <a
            className="navbar-item"
            href="/best"
            onClick={[Function]}
          >
            Hacker news
          </a>
          <div
            className=" navbar-burger burger"
            data-target="main-navbar"
            onClick={[Function]}
          >
            <span />
            <span />
            <span />
          </div>
        </div>
        <div
          className=" navbar-menu"
          id="main-navbar"
        >
          <div
            className="navbar-end"
          >
            <a
              className="navbar-item"
              href="/best"
              onClick={[Function]}
            >
              Best
            </a>
            <a
              className="navbar-item"
              href="/news"
              onClick={[Function]}
            >
              News
            </a>
            <a
              className="navbar-item"
              href="/show"
              onClick={[Function]}
            >
              Show
            </a>
            <a
              className="navbar-item"
              href="/ask"
              onClick={[Function]}
            >
              Ask
            </a>
            <a
              className="navbar-item"
              href="/jobs"
              onClick={[Function]}
            >
              Jobs
            </a>
          </div>
        </div>
      </nav>
    </div>
    <header
      className="hero is-link"
    >
      <div
        className="hero-body"
      >
        <div
          className="container has-text-centered"
        >
          <h1
            className="title is-capitalized"
          />
        </div>
      </div>
    </header>
  </div>
  <div
    className="section"
  >
    <div
      className="container"
    />
  </div>
  <div
    className="section"
  >
    <div
      className="container"
    >
      <div
        className="columns"
      >
        <div
          className="column is-2"
        >
          <nav
            aria-label="pagination"
            className="pagination is-right"
            role="navigation"
          />
        </div>
      </div>
    </div>
  </div>
  <footer
    className="footer"
  >
    <div
      className="container"
    >
      <div
        className="content has-text-centered"
      >
        <p>
          <strong>
            Hacker News
          </strong>
          <br />
          Copyright Â© - 
          2018
        </p>
      </div>
    </div>
  </footer>
</div>
`;
